packages/example/src/ThreeVideo/index.tsx

```diff
@@ -0,0 +1,20 @@
+import {ThreeCanvas, ThreeVideo} from '@remotion/three';
+import React from 'react';
+import {Sequence} from 'remotion';
+
+const videoSrc =
+	'http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4';
+
+const Video = (): React.ReactElement => {
+	return (
+		<div style={{display: 'flex', width: '100%'}}>
+			<ThreeCanvas>
+				<Sequence layout="none" from={-460} durationInFrames={Infinity}>
```

Comment:
The sequence does still throw some errors if you play around with it (parent not found). I will investigate this